gulp-core-build-typescript/src/RunApiExtractorOnExternalApiTypes.ts

```diff
@@ -0,0 +1,37 @@
+import * as gulp from 'gulp';
+import * as path from 'path';
+import * as fs from 'fs';
+
+import { GulpTask } from '@microsoft/gulp-core-build';
+import { ExternalApiHelper } from '@microsoft/api-extractor';
+
+const files: string[] = ['external-api-types/es6-collections/index.d.ts',
+                         'external-api-types/es6-promise/index.d.ts',
+                         'external-api-types/whatwg-fetch/index.d.ts'];
+
+export class RunApiExtractorOnExternalApiTypes extends GulpTask<void> {
+  public name: string = 'run-api-extractor';
+
+  public executeTask(gulp: gulp.Gulp, completeCallback?: (result?: Object) => void): void {
+    let outputPath: string = path.join(this.buildConfig.rootPath, this.buildConfig.distFolder);
+    if (!fs.existsSync(outputPath)) {
+      fs.mkdirSync(outputPath);
```

Comment:
If this is the last step of the build, surely the "dist" folder would already exist, right?